db/test/unit/tables/ccbc_user_test.sql

```diff
@@ -116,5 +116,13 @@ select throws_like(
     'ccbc_guest cannot delete rows from table_ccbc_user'
 );
 
+select throws_like(
+  $$
+    select * from ccbc_public.applications
+  $$,
+  'permission denied%',
+    'ccbc_guest cannot select rows from table_applications'
+);
+
```

Comment:
can we move this to a new `application_test.sql` file? 
We will want to test the other permissions as well (insert, update, delete)

Reply:
Moved tests to `ccbc_guest_test.sql`